internal/cmd/load-balancer/observability-credentials/list/list.go

```diff
@@ -39,6 +44,9 @@ func NewCmd(p *print.Printer) *cobra.Command {
 			examples.NewExample(
 				`List all observability credentials for Load Balancer`,
 				"$ stackit load-balancer observability-credentials list"),
+			examples.NewExample(
+				`List all used observability credentials for Load Balancer`,
```

Comment:
I think we could make the example more clear by writting something like `List all observability credentials being used by a Load Balancer`

Reply:
And maybe rewrite other examples as `List all Load Balancer observability credentials`